packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.scss

```diff
@@ -28,7 +28,7 @@
     &-field {
         margin-bottom: 0 !important;
         flex: 1;
-        height: 4rem;
```

Comment:
```scss
min-height: 4rem;
```

Reply:
just a suggestion, ignore it if not needed

Reply:
i dont think its needed. tried it. couldn't find any difference.